index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/fddcccfd38.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Local TubeClone</title>
+</head>
+
+<body>
+    <!--header start-->
+    <header>
+        <nav class="flex-container">
+            <i class="fas fa-bars"></i>
+            <img class="logo" src="images/logo" alt="image of logo">
+            <form action="">
+                <input class="myInput" type="text" id="search" placeholder="Search"><i class="fas fa-search"></i>
+            </form>
+
+            <i class="fas fa-upload"></i>
+            <button class="sign" type="submit">Sign in</button>
```

Comment:
- [ ] Although you did a great job cloning the sample page, you should also take care of Microverse'sspecial requirements in the Pathwright, please check that you need at least 3 embedded images in this part
![image](https://user-images.githubusercontent.com/53256139/91040000-2469e000-e5d3-11ea-83f8-791804c6c92f.png)